install_swiftformat.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Installs the SwiftFormat package.
+# Tries to get the precompiled .pkg file from Github, but if that
+# fails just recompiles from source.
+
+set -e
+
+brew update && brew install swiftformat
+
+if ! type swiftformat > /dev/null 2>&1; then
+    echo "SwiftFormat package doesn't exist. Compiling from source..." &&
```

Comment:
not to nitpick error messages, but might be worth mentioning that it doesn't exist after an attempt to install it via brew